validator/src/state/merkle.rs

```diff
@@ -628,6 +743,20 @@ mod tests {
                 "Should not contain the value"
             );
 
+            let change_log: ChangeLogEntry = {
+                // check that we have a change log entry for the new root
+                let reader = db.reader().unwrap();
+                protobuf::parse_from_bytes(&reader
+                    .index_get(CHANGE_LOG_INDEX, new_root_bytes)
+                    .expect("No db errors")
+                    .expect("A change log entry"))
+                    .expect("The change log entry to have bytes")
```

Comment:
expect should take an error message, not what you expect

Reply:
Base on the previous comment I will update this, but in other instances I've used functions like  'expect', the whole statement reads like a sentence: Expect no db errors, expect a change log entry, etc. But the way it is printed (which I don't particularly) looks like it loses this context.